src/index.html

```diff
@@ -1,26 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>Knockout TypeScript</title>
-
-  <link
-      rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
-      crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 
 </head>
-<body>
-<nav class="navbar navbar-dark bg-dark">
-  <div class="container-fluid">
-    <a class="navbar-brand" href="#">
-      <img src="assets/images/image2vector.svg" alt="" width="30" height="24" class="d-inline-block align-text-top">
-      CRM
-    </a>
-  </div>
-</nav>
 
+<body id="app">
+  <nav class="navbar navbar-dark bg-dark">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="#">
+        <img src="assets/images/image2vector.svg" alt="" width="30" height="24" class="d-inline-block align-text-top">
+        CRM
+      </a>
+      <button data-bind="click: version" class="btn btn-light my-2 my-sm-0" type="button" data-toggle="modal"
+        data-target="#exampleModal">version</button>
+    </div>
+
+  </nav>
+  <modal-version></modal-version>
+
+
+
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
```

Comment:
@DianaVargasOrellana, I did not see any place where you are using JQuery, furthermore, I think we should not use JQuery since all things that we need are already provided by knockout and Axios. So, if we are not using a library, we should remove it from our dependencies.

Reply:
I am using jquery to display the modal